.github/label_actions.yaml

```diff
@@ -0,0 +1,35 @@
+assume-fixed:
```

Comment:
These should be added to our bot instead

Reply:
I've opened the pull request, https://github.com/home-assistant/service-hub. It took me a while to figure out where, when and how based on that comment.